sentry/src/main/java/io/sentry/Hub.java

```diff
@@ -616,16 +615,19 @@ public void flush(long timeoutMillis) {
   }
 
   @Override
-  public void setSpanContext(
-      final @NotNull Throwable throwable, final @NotNull SpanContext spanContext) {
+  public void setSpanContext(final @NotNull Throwable throwable, final @NotNull ISpan span) {
```

Comment:
This method deserves a better name now. I am open for suggestions.

Reply:
naming is hard, off by 1 errors too :)

should it be `@ApiStatus.Internal`?

Reply:
It is actually on the interface, I'll add it to implementation too. 👍